### DISCUSSION TITLE
Sign up for the Hackathon + Introduce yourself!

### DISCUSSION THREAD

**Discussion by sudokita** [ag-informatics/hackag.tech#1] [HACKAG.TECH 2024 Discussion Forum!]
Hi folks,

In preparation for our digital agriculture hackathon, HackAg.Tech @ [ASABE Hackathon 2024](https://www.asabemeetings.org/), let's get to know each other!

**Who am I**
I'm Ankita Raturi (she/her) an assistant professor from Purdue University, where I run an [Agricultural Informatics lab](http://aginformaticslab.org/). I've been involved in organizing this hackathon since 2021 and we're looking forward to growing this into a more hands-on event in the lead up and I teach courses related to informatics for food, agriculture, and the environment. This year @joedvorak and @joseph will be taking lead on our organizing committee!

**Skills that I'm comfortable with**
Python and enough of a few others that I can help out
Web development using Django, but can debug javascript or php if you need.
Some IoT basics.
Software Design is my jam!
Databases development including data modeling.
Spreadsheet wrangling (just not as good as Dennis!)
Data Science pipeline & ML fundamentals 

**What I'd like to learn at the hackathon**
I'd love to learn how to improve the hackathon to support more diverse ways of participating! Anyone and everyone can participate in hackathons, so I'd like to figure out how to grow out the experience to help folks with skills other than coding to also feel right at home :)

**What I'll bring to the hackathon**
I'll bring data, ideas, and a whole lotta good vibes!

**Hackathon Project Ideas**
I am an organizer / mentor. I'll be there to help with all of the projects! If you're looking for ideas, I can help :) 

**Your turn!**
|
|
|
|
|
### Template Reponse
**Please copy and answer these questions in your response!** 
* **Who am I** e.g., preferred name, pronouns, affiliations. Are you student? job? department? topic area?
* **Skills that I'm comfortable with:** (select from and describe as you see fit)
    * Python, C, or any other Programming Language
    * Javascript, Django, or any other Web Development Languages/Frameworks
    * R, Matlab, or any other Scripting Language
    * IoT skills , e.g., working with microcontrollers, sensors, programming
    * Software Design, e.g., mapping user flow, creating mockups, wireframes, and graphic materials
    * Database development skills, e.g., using SQL
    * Spreadsheet wrangling, incl. writing complex functions
    * Data science and/or Machine learning skills
    * Other (tell us more!)
* **What I'd like to learn at the hackathon** (Describe in <250 words, bullets OK!)
* **What I'll bring to the hackathon**? (Examples include: data, hardware, software, written texts, and positive vibes!)
* **Hackathon Project Ideas**
    * I am looking for a project related to [insert topics]
    * I have a project idea [insert little outline of a project idea] --> once you've found a few people. Start a new discussion thread like this: [link to ankita example]
    * I am an organizer / mentor / other
* Anything else you'd like us to know about you

**Comment by jquansah** [comment]
**Who am I**
I’m Joseph Quansah, Associate Professor and Director of Geospatial and Climate Change Center at Tuskegee University.  I have been part of the USDA NIFA HEC grant program since 2021. This will be my second participation at the hackathon at AIMs. I am one of the lead organizers for the 2024 Plant Breeding and Data Analytics Hackathon at ARD Meeting in Nashville, TN, and hope to co-leadwith @joedvorak in organizing this year’s hackathon.

**Skills that I'm comfortable with**
I have some good knowledge of python and Matlab, but wouldn’t say I am an expert in these areas.
Spatial modular scripting with python,  Machine and deep learning learning applications in image processing
The use of various geospatial and image processing software.

**What I'd like to learn at the hackathon**
I would like to learn how best to get more minority students to participate in this and other hackathons to help enhance their data science skills and AI applications in Agriculture.
What I'll bring to the hackathon
I hope to bring all the support and encourage that all participants will need to enjoy and be successful at the hackathon.

**Hackathon Project Ideas**
I am an organizer / mentor and will help participant with deciding on their project ideas and solutions.
